rosidl_generator_py/cmake/rosidl_generator_py_get_typesupports.cmake

```diff
@@ -15,7 +15,7 @@
 macro(accumulate_typesupports)
   set(_typesupport_impl "")
   get_rmw_typesupport(_typesupport_impl ${rmw_implementation} LANGUAGE "C")
-  list(APPEND _typesupport_impls ${_typesupport_impl})
+  list_append_unique(_typesupport_impls ${_typesupport_impl})
```

Comment:
Already an existing problem in this file: https://github.com/ros2/rosidl/pull/156/files#diff-91406625e1eac1b288db266065a388a5L26

I'll add the export to this package in this PR.